docs/upgrade-to-v2.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[upgrade-to-v2]]
+
+ifdef::env-github[]
+NOTE: For the best reading experience,
+please view this documentation at https://www.elastic.co/guide/en/apm/agent/nodejs/current/upgrade-to-v2.html[elastic.co]
+endif::[]
+
+=== Upgrade to v2.x
+
+The following is a guide on upgrading your Node.js agent from version 1.x to version 2.x.
+
+[[v2-overview]]
+==== Overview
+
+Version 2.0.0 of the Node.js agent requires version 6.5 of the APM Server or higher and makes use of the new HTTP intake API.
+
+The new agent supports Node.js 6, 8, and 10+.
+
+The format of the error ID's have changed from a UUID4 to a hex formatted random 128 bit number.
+Likewise,
+the format of the transaction ID's have changed from a UUID4 to a hex formatted random 64 bit number.
+
+[[v2-config-options]]
+==== Config options
+
+[[v2-configuration-order]]
+===== Configuration order
+
+The Node.js agent can be configured using a combination of inline config options,
+environment variables,
+and a config file.
+Many config options also have default values.
+The order in which all these are applied have changed in 2.0.0.
+
+In 1.x the order in which config options overruled each other was (higher overwrite lower):
+
+- Inline options given to <<apm-start,`.start()`>>
+- <<agent-configuration-file,Agent config file>>
+- Environment variables
+- Default values
+
+The new order in 2.0.0 is as follows (higher overwrite lower):
+
+- Environment variables
+- Inline options given to <<apm-start,`.start()`>>
+- <<agent-configuration-file,Agent config file>>
+- Default values
+
+[[v2-changed-units]]
+===== Changed units
+
+In 1.x,
+the config option <<aborted-error-threshold,`abortedErrorThreshold`>> expected a millisecond value.
+In 2.0.0 the default time unit is seconds.
+If you like to keep using milliseconds,
+you need to specify the unit,
+e.g: `1500ms`.
+
+In 1.x,
+all boolean config options could be configured using the strings `on`,
+`yes`,
+`1` etc to mean `true` with similar values representing `false`.
+In 2.0.0 this have been restricted,
+and only the strings `true` and `false` will be interpreted as the boolean equivalent.
+
+[[v2-removed-config-options]]
+===== Removed config options
+
+The following config options have been removed in version 2.0.0:
+
+|=======================================================================
+|Name |Note
+|`flushInterval` |Use <<api-request-time,`apiRequestTime`>> instead. Note that this option have a slightly different meaning as the intake API have changed.
```

Comment:
```suggestion
|`flushInterval` |Use <<api-request-time,`apiRequestTime`>> instead. Note that this option has a slightly different meaning as the intake API has changed.
```

Reply:
No worries, that's what I'm here for! A general trick that might help you is "has" is typically singular, and "have" is typically plural. So:

It/this => has
they/these => have

_Wipes off chalkboard_